frontend/src/components/datasources/components.tsx

```diff
@@ -30,3 +30,11 @@ export const EmptyState: React.FC<{ content: string; className?: string }> = ({
     </div>
   );
 };
+
+export const ItemSubtext: React.FC<{ content: string }> = ({ content }) => {
+  return (
+    <span className="text-xs text-black bg-gray-200 rounded px-1">
```

Comment:
`bg-gray-200` won't support dark mode, and i think it could be slightly lighter

maybe `bg-[var(--slate-8] text-[var(--slate-9]`?